### PR TITLE
Added hosts:insert command

### PR DESCRIPTION
**Pull request by rimi-itk**

### CODE REVIEW

File: scripts/itkdev-docker-compose [hunk 1]
```diff
@@ -84,6 +86,14 @@ case "$cmd" in
     docker-compose up -d
     ;;
 
+  hosts:insert)
+    DNS=${COMPOSE_PROJECT_NAME}.docker.localhost
+    # Remove any existing entry
+    sudo sed -i '.itkdev-docker-compose' "/$DNS/d" /etc/hosts
+    # Append the entry
+    echo "127.0.0.1 $DNS # itkdev-docker-compose" | sudo tee -a /etc/hosts
```
Comment (cableman) [inline scripts/itkdev-docker-compose]: Should that have been 0.0.0.0 or is that the same?

Comment (rimi-itk) [reply scripts/itkdev-docker-compose]: Using `127.0.0.1` works, but I was thinking the same thing (“Should it be 0.0.0.0?”)

Comment (cableman) [reply scripts/itkdev-docker-compose]: Docker just uses the `0.0.0.0`in all place... 

Look at https://stackoverflow.com/questions/20778771/what-is-the-difference-between-0-0-0-0-127-0-0-1-and-localhost

I think `0.0.0.0` might be more correct in this context.. but it's up to you

Comment (rimi-itk) [reply scripts/itkdev-docker-compose]: I'll change it.

**Pull request merged**